guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -47,17 +47,43 @@ Apache user must not have access to the keytab file.
 ----
 [service/HTTP]
 mechs = krb5
-cred_store = keytab:/etc/httpd/conf/http.keytab
+cred_store = keytab:/etc/gssproxy/http.keytab
 cred_store = ccache:/var/lib/gssproxy/clients/krb5cc_%U
 euid = __ID_of_Apache_User__
 ----
+. Set the correct SELinux context for the future keytab entry:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# semanage fcontext -a -s system_u -t krb5_keytab_t '/etc/gssproxy/http\.keytab'
+----
 . Create a keytab entry:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# KRB5_KTNAME=FILE:/etc/httpd/conf/http.keytab net ads keytab add HTTP -U administrator -d3 -s /etc/net-keytab.conf
-# chown root.{apache-user} /etc/httpd/conf/http.keytab
-# chmod 640 /etc/httpd/conf/http.keytab
+# KRB5_KTNAME=FILE:/etc/gssproxy/http.keytab net ads keytab add HTTP -U administrator -d3 -s /etc/net-keytab.conf
+# chown root.root /etc/gssproxy/http.keytab
+# chmod 600 /etc/gssproxy/http.keytab
+----
+// TO BE CONFIRMED: This step should not be needed
+. Add a *Service Principal Name (SPN)* for your AD server.
+You can use a tool such as `adcli`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# adcli spn --add _My_Service_/{foreman-example-com} _My_Username_
+----
+. Make sure the following line is included in the `/etc/sssd/sssd.conf` file:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+ad_gpo_map_service = +foreman
+----
+. If you changed the `/etc/sssd/sssd.conf` configuration file, restart the sssd service:
```

Comment:
```suggestion
. If you changed the `/etc/sssd/sssd.conf` configuration file, restart the `sssd` service:
```
It's either "the SSSD service" or "the `sssd` service". And other steps reference services in the format of "the `httpd` service" etc.